### ISSUE TITLE
[Crash] Exception ticking world, net.minecraft.server.management.PlayerManager

### ISSUE THREAD

**Issue by vadimyer** [CyberdyneCC/Thermos#364]
**Server Log:** *(Use [Pastebin](http://pastebin.com/) or [Github Gist](https://gist.github.com/))*
http://pastebin.com/300xscuJ

**FML Log(s):** *(Use [Pastebin](http://pastebin.com/) or [Github Gist](https://gist.github.com/))*
I'll post if you need.

**Explanation of issue:**
Just a crash report, I don't really know what happened.

**How to recreate this issue:**
Don't know.

**Modpack Name:** *(Only if public)*
Custom, see the report.

**Mods Installed:**
See the report.

**Plugins Installed:**
I'll post them if you need, pretty standard ones.

**Warmroast Report:** *(Optional)*
-

**Thermos Version:**
B55

**Forge Version:**
1614

**Comment by Robotia** [comment]
Ok I looked at the error...somehow it got a null chunk from the world that the player was in? This should not be possible normally, did you do anything before it happened? Also, DIM1 is the End right?

**Comment by vadimyer** [comment]
DIM1 is the end, right. Unfortunately I'm not sure what caused this :( I didn't do anything with the server at the time it happened and before.

**Comment by iKaneki-Ken** [comment]
@Robotia It more than likely has to do with GalactiCraft. It's been known to screw with things like this.

**Comment by Robotia** [comment]
That is true, I remember there was an issue earlier concerning Galacticraft but not sure if it got resolved :confused:
